coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -52,26 +50,28 @@ import java.io.File
 interface ImageLoader {
 
     /**
-     * The default options that are used to fill in unset [Request] values.
+     * The default options that are used to fill in unset [ImageRequest] values.
      */
     val defaults: DefaultRequestOptions
 
     /**
-     * Launch an asynchronous operation that executes the [LoadRequest] and sets the result on its [Target].
+     * Execute the [request] asynchronously.
```

Comment:
Is it confusing to use the term `execute` in the docs of the `enqueue` function?

Reply:
Good point, updated the docs.